### PR TITLE
changing early debugger exit to terminal highlight (#8)

### LINKED ISSUE #8: Logs don't tail after "Brightscript Debugger>" appears in console output
After experiencing a crash (or legitimately using the BrightScript debugger via `telnet ip.add.re.ss 8085`), the first few installed builds don't result in logs getting printed.  It appears that connecting to a Roku over `telnet` (like this CLI does) always returns a few lines of history.

Any way to make this not happen?  I wind up `^C`ing the script after it installs and just running `telnet` directly, so something like `roku --install-only ip.add.re.ss && telnet ip.add.re.ss 8085` would be lovely.  Or even just letting the logs continue to flow after detecting `Brightscript Debugger>`?

### PR DESCRIPTION
**Pull request by strattonbrazil**
I removed the code that forced the early exit from previous runs, but I thought it was still valuable to see highlighting to signal the debugger was present.  I can remove that if it doesn't seem valuable.  

example_coloring

closes #8

### CODE REVIEW

File: roku_dev_cli/roku_dev_cli.py [hunk 1]
```diff
@@ -116,16 +125,17 @@ def run(self):
...
-                    if text.rfind("Brightscript Debugger>") != -1:
-                        print(bcolors.FAIL + "Press any key to exit" + bcolors.ENDC)
-                        exit(1)
-
+                            if self.timestamp:
+                                timestamp = datetime.datetime.now().strftime('%h %d, %Y %I:%M:%S%p')
+                                line = bcolors.OKBLUE + "[" + timestamp + "] " + bcolors.ENDC + line
+                            
+                            # add red to highlight debugger output
+                            if line.strip() in "".join(debuggerHighlights):
```
Comment (sjbarag) [inline roku_dev_cli/roku_dev_cli.py]: Consider performing this `join` once at the top (near where `debuggerHighlights` is defined) – it looks like we'd join the same lines together for every read telnet line.

Actually, do we even need to do a join?  AFAIK `set` still allows the `in` operator, so `if line.strip() in debuggerHighlights` might be sufficient!

**Review by sjbarag — COMMENTED** [review]
Looks good overall!  I'm curious if there's a way we could avoid string-joins in such a tight loop.

File: roku_dev_cli/roku_dev_cli.py (hunk 1 above)
Comment (strattonbrazil) [reply roku_dev_cli/roku_dev_cli.py]: Durp.  Yes, I think I had this before I switched to using line strip.  Don't remember why, but obviously it wasn't a good idea.  Will change.

**Review by sjbarag — APPROVED** [review]
I could have sworn I'd clicked "approve" last time?  Anyway, looks wonderful!

**Pull request merged**